test/src/tests/acceptancetests/lrogrouptest/lro_test.go

```diff
@@ -6,10 +6,11 @@ import (
 	"testing"
 	"time"
 
+	"github.com/Azure/go-autorest/autorest/to"
 	chk "gopkg.in/check.v1"
 
 	"tests/acceptancetests/utils"
-	. "tests/generated/lro"
```

Comment:
>.  [](start = 1, length = 2)

i thought we run go fmt part of CI and check if the pushed content is the same as the resulting one ?

Reply:
Importing a package with a preceding "dot" means to import its content into the importing package's namespace.  Unfortunately this breaks stuff in vscode (e.g. right-clicking on a func to jump to its definition doesn't work) so I removed the "dot" from the import and reference the package content explicitly.

Reply:
Oh my bad, i thought it was a whitespace ( since whitespaces are rendered as dots in CodeFlow ) :)